src/Perpetuum/Zones/Teleporting/Strategies/TrainingExitStrategy.cs

```diff
@@ -16,6 +18,7 @@ public class TrainingExitStrategy : ITeleportStrategy
     {
         private const double CHARACTER_START_CREDIT = 500000; //TODO: move to DB
         private const int MAX_REWARD_LEVEL = 4;
+        private const int WAIT_TIME_BEFORE_SENDING_MAIL_MS = 10000;
```

Comment:
Does this delay (10 seconds) seems reasonable?
With the tests I've done, the user will see "New Mail" notification a few seconds after the welcome window shows up.
The feedback of "New Mail" notification for the user is the same for the initial email from the character creation.

Reply:
Same comment as above in `CharacterCreate`
For the actual duration - the best feedback will be from the players at the end of the day.  I trust your judgement on the value to start with.